From 97a12ede84730737fd1ff974e58be00b06585b15 Mon Sep 17 00:00:00 2001
From: Tim Stahlhut <stahta01@gmail.com>
Date: Sun, 13 Dec 2020 01:55:55 -0500
Subject: Parsing: fix segmentation fault

Verify both sub pointers are good before using them.
The segfault happened using the MSys2 (Cygwin like) enviroment
under Windows 7 64 bit.
---
 src/parse-util.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/parse-util.c b/src/parse-util.c
index 9c6da94aa..0ce9ca94d 100644
--- a/src/parse-util.c
+++ b/src/parse-util.c
@@ -654,7 +654,7 @@ parsepos_as_string (GnmParsePos const *pp)
 
 	return cell_coord_name2 (pp->eval.col,
 				 pp->eval.row,
-				 pp->sheet && pp->sheet->convs->r1c1_addresses);
+				 pp->sheet && pp->sheet->convs && pp->sheet->convs->r1c1_addresses);
 }
 
 char const *
-- 
